projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-token-subscription-service.php

```diff
@@ -218,7 +218,7 @@ protected function validate_subscriptions( $valid_plan_ids, $token_subscriptions
 		foreach ( $token_subscriptions as $product_id => $token_subscription ) {
 			if ( in_array( $product_id, $product_ids, true ) ) {
 				$end = is_int( $token_subscription->end_date ) ? $token_subscription->end_date : strtotime( $token_subscription->end_date );
-				if ( $end > time() ) {
+				if ( ( $end + 24 * 3600 ) > time() ) {
```

Comment:
Oh, just FYI there is a WordPress constant called DAY_IN_SECONDS but for myself I'm pretty sure every dev knows how many seconds are in a day or hour...